prov/efa/src/efa_user_info.c

```diff
@@ -610,6 +610,16 @@ int efa_getinfo(uint32_t version, const char *node,
 	struct fi_info *dgram_info_list, *rdm_info_list;
 	int err;
 
+#ifndef _WIN32
+	err = efa_fork_support_install_fork_handler();
```

Comment:
Nit: since we are not installing it during provider init, let's add back that TODO here like

```
TODO:
It'd be better to install this during provider init (since that's
only invoked once) but fork() is currently called by nvml_init in hmem_hook provider's init
which is called after efa provider init. This can move to the provider init after we get rid of
that fork() in other provider's init.
```